eng/targets/GenerateAnalyzerNuspec.targets

```diff
@@ -111,8 +111,12 @@
       <AnalyzerRulesetAssembly Update="@(AnalyzerRulesetAssembly)" Condition="'%(AnalyzerRulesetAssembly.TargetFramework)' == ''" TargetFramework="$(TargetFramework)" />
     </ItemGroup>
 
+    <PropertyGroup>
+      <SkipDocumentationValidation>true</SkipDocumentationValidation>
+    </PropertyGroup>
+
     <!-- Only run validate only in CI builds. Running them in local builds will prevent refreshing auto-generated files. -->
-    <Exec Condition="'$(ContinuousIntegrationBuild)' == 'true'"
+    <Exec Condition="'$(ContinuousIntegrationBuild)' == 'true' and '$(SkipDocumentationValidation)' != 'true'"
```

Comment:
Wonder if a more ideal check would be, CI but not an official build.

Reply:
CI is the part we are trying to unblock (temporarily).  Need to figure out some alternative that doesn't take CI down if a help link uri is invalid or temporarily down.